submissions/Yasya23/holi-style-popup/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Menu</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <nav class="navigation">
+      <ul class="navigation__list">
+        <li class="navigation__item"><a href="#">Mail</a></li>
+        <li class="navigation__item"><a href="#">Images</a></li>
+        <li class="navigation__item popup">            
+            <label for="popup"
+            ><img
+              src="icons/popup-button.png"
+              alt="Popup menu icon"
+              width="28"
```

Comment:
This decision is not scalable, every time you will need to specify dimensions of image. Try to play with CSS properties, e.g. `object-fit`

Reply:
I indicated the size since a lot of places write that this is necessary for pictures. In CSS, I defined the max width and height so that the layout does not break. As in the example, the icon does not exceed the limits of the parent container, regardless of the size.

![Img size](https://github.com/Yasya23/Hooli-style-Popup/blob/main/img/img.jpg)

Reply:
The image did not load, most likely due to the size, but it is active on the link